Jenkinsfile_CNP

```diff
@@ -12,6 +14,8 @@ def secrets = [
   ]
 ]
 
+env.PACT_BROKER_FULL_URL = 'https://pact-broker.platform.hmcts.net'
```

Comment:
Don't think u need this.

Reply:
Jenkins maintains the URL https://github.com/hmcts/cnp-jenkins-library/blob/9b4e3c83ce173733ca19c61ecd5f8ddadf0fff5e/vars/sectionBuildAndTest.groovy#L197 , maintaining on our side means we need to make a change when platops do.